index.html

```diff
@@ -221,7 +221,6 @@ <h1>Uber Navigation</h1>
                         <li>
                            <p>javascript</p> 
                         </li>
-
```

Comment:
- [ ] please ensure that switching from your mobile to desktop version is working smoothly(responsive). your mobile version cards are showing one-sided. 
![Screenshot (111)_LI](https://user-images.githubusercontent.com/65298048/172565777-5a1a34e8-643f-4e74-b293-f8d334869a4d.jpg)